### DISCUSSION TITLE
Onnxruntime Python vs ML.NET Library Version

### DISCUSSION THREAD

**Discussion by gjiremadzeBECKMAN** [microsoft/onnxruntime#10717] [Other Q&A] [answered]
Hey All,

I was wondering if anyone knows what the current onnxruntime library version is exactly? Or more specifically I would like to know what python onnxruntime version matches up with the current onnxruntime that's in ML.NET. 

I currently am dealing with a keras model that I am generating in Python that has a custom layer. The onnx model is able to load and perform inference when using it in python. However, when loading the model in C# ML.NET I'm getting a 'Protobuf Parsing Failed" exception. 

I would like to get the same versioned onnxruntime library in python that's currently in ML.NET to see if this error exists there as well. 

Thank you

**Comment by snnn** [comment]
I don't know much on ML.NET. Onnxruntime has nuget packages on nuget.org: https://www.nuget.org/packages?q=onnxruntime . They are published together with the python packages when we do a release. Their version match. ML.NET is a different product, they may have their own release schedule that I'm not familiar with.

**Comment by luisquintanilla** [comment, marked as answer]
Adding this thread for additional context dotnet/machinelearning#6098

The current ONNX runtime version in ML.NET is 1.6.
